otter/models/intents.py

```diff
@@ -78,6 +78,25 @@ def perform_update_group_status(dispatcher, ugs_intent):
 
 
 @attr.s
+class UpdateScalingGroupStatus(object):
```

Comment:
This class name confused me at first.  The salient property here is not "scaling", as "group" and "scaling group" are synonyms, but rather the fact that one is *loaded already* and one isn't.  Do you think a name-change here to `LoadAndUpdateGroup` would make sense?  Given that there's an "and" in the name, is there any way to more gracefully compose the group-loading and existing group-updating effects?  (If not, that's fine, but if we could get away with fewer effects that need distinct performers, that would be great.)